src/Components/BeingStockholmBeta.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react'
+import { Header, Container, Image, List, Segment, Grid, Sidebar, Divider } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+
+class BeingStockholmBeta extends Component {
+  render() {
+    return (
+      <>
+        <Sidebar.Pushable as={Container} id="views-main-container-sidebar">
+          <div onClick={this.props.sidebarVisible ? () => { this.props.dispatch({ type: 'CHANGE_VISIBILITY' }) } : () => { }}>
+            <Container className="views-main-container">
+              <Header className="views-main-header" as='h1'>
+                Being Stockholm Beta
+              </Header>
+              <br></br>
+
+              <Container className="views-ingress-container">
+                <p>
+                This is our first web app to test the project idea over Summer 2019.
+                </p>
+              </Container>
+              <br></br>
+
+              <Container className="views-text-container">
+                <p>
+                  We are extremely grateful to the coaches and senior students at the <b><a href='https://craftacademy.se/english/' target='_blank' rel='noopener noreferrer'>Craft Academy</a></b> Coding Bootcamp who helped us to develop our concept into a beautiful working prototype as part of their final design project.
```

Comment:
should we style the clickable links so that theyre in the brand colour? (designer here again, ignore this if you think it doesnt matter)

Reply:
done